Content.Server/Explosion/EntitySystems/ExplosionSystem.Processing.cs

```diff
@@ -434,16 +440,38 @@ private void GetEntitiesToDamage(EntityUid uid, DamageSpecifier originalDamage,
         float throwForce,
         string id,
         TransformComponent? xform,
-        float? fireStacksOnIgnite)
+        float? fireStacksOnIgnite,
+        EntityUid? explosionCause // RMC-14
+        )
     {
         if (originalDamage != null)
         {
             GetEntitiesToDamage(uid, originalDamage, id);
             foreach (var (entity, damage) in _toDamage)
             {
+                // BEGIN RMC-14 ADDITION: Log explosion damage to players.
+                if (TryComp<ActorComponent>(entity, out var actorComponent)) // Filter for "humanoids".
+                {
+                    if (explosionCause == null)
+                    {
+                        _adminLogger.Add(
+                            LogType.HitScanHit,
```

Comment:
This should probably be a different or new log type for all of these, unless you think this is better as an admin then that works